src/bun.js/webcore/blob.zig

```diff
@@ -708,14 +708,43 @@ pub const Blob = struct {
     ) JSC.JSValue {
         const destination_type = std.meta.activeTag(destination_blob.store.?.data);
 
-        // Writing an empty string to a file is a no-op
         if (source_blob.store == null) {
-            destination_blob.detach();
-            return JSC.JSPromise.resolvedPromiseValue(ctx.ptr(), JSC.JSValue.jsNumber(0));
+            return switch (destination_type) {
+                .file => {
+                    // when writing empty blob to file, we just create an empty file
```

Comment:
Since this is creating an empty file, we only need to do `openat(path, O_TRUNC | O_CREAT)`

Reply:
we could probably reuse `writeStringToFileFast` here, similar to how it is used earlier in `Blob.writeFile`

Reply:
Hi. Thanks for the advice, it's very helpful!
I refactored it to use `writeStringToFileFast`! What do you think?